pkg/api/api.go

```diff
@@ -57,6 +57,7 @@ var apiMap = map[string]string{
 	"conditional-naming-host":         "/api/config/v1/conditionalNaming/host",
 	"conditional-naming-service":      "/api/config/v1/conditionalNaming/service",
 	"maintenance-window":              "/api/config/v1/maintenanceWindows",
+	"request-naming":                  "/api/config/v1/service/requestNaming",
```

Comment:
Should we use a folder name which has `service` in its name? E.g. `request-naming-service`?

This way we would indicate that its a service api. Furthermore, if there will be a request naming api for a different type of entity in the future, it would be less confusing. Its then also aligned with the other folder names like `conditional-naming-service`.

What do you think?